core/lib/state/src/rocksdb/mod.rs

```diff
@@ -8,18 +8,23 @@
 //! - Contracts
 //! - Factory dependencies
 //!
-//! | Column       | Key                    | Value                   | Description                          |
-//! | ------------ | ---------------------- | ----------------------- | ------------------------------------ |
-//! | State        | 'block_number'         | serialized block number | Last processed L1 batch number (u32) |
-//! | State        | hashed `StorageKey`    | 32 bytes value          | State for the given key              |
-//! | Contracts    | address (20 bytes)     | `Vec<u8>`               | Contract contents                    |
-//! | Factory deps | hash (32 bytes)        | `Vec<u8>`               | Bytecodes for new contracts that a certain contract may deploy. |
-
-use std::{collections::HashMap, mem, path::Path, time::Instant};
+//! | Column       | Key                             | Value                           | Description                               |
+//! | ------------ | ------------------------------- | ------------------------------- | ----------------------------------------- |
+//! | State        | 'block_number'                  | serialized block number         | Last processed L1 batch number (u32)      |
+//! | State        | 'enum_index_migration_cursor'   | serialized hashed key           | Hashed key migration should continue from |
+//! | State        | 'enum_index_migration_finished' | 1 byte flag                     | Denotes if the migration if completed     |
```

Comment:
Rly smol nit: Technically, the *existence* of this key denotes that a migration is completed, and the value doesn't matter. (One could think that 0 value means that the migration is not completed, but this is not the case.) On that note, IMO it would be easier to combine cursor with the flag (e.g., use an empty cursor bytes to denote that the migration is complete), but that's just a personal preference.

Reply:
I would also combine but yeah it's very minor